staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3324,6 +3324,9 @@ type PodCondition struct {
 	// Human-readable message indicating details about last transition.
 	// +optional
 	Message string `json:"message,omitempty" protobuf:"bytes,6,opt,name=message"`
+	// The generation of the pod observed at the point this condition is being reported.
+	// +optional
+	ObservedGeneration int64 `json:"observedGeneration,omitempty" protobuf:"varint,8,opt,name=observedGeneration"`
```

Comment:
I'm not sure if there is a protocol for how to pick the field numbers for new fields. The ones I picked are arbitrary.